core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -405,28 +394,40 @@ def _initialize(self, X=None, y=None, feature_metadata=None, num_classes=None, *
             if self.num_classes is None:
                 label_cleaner = LabelCleaner.construct(problem_type=self.problem_type, y=y)
                 self.num_classes = label_cleaner.num_classes
+
+        self._init_params_aux()
```

Comment:
It may be helpful for the `_init_params_aux()` method to also have docstring that briefly mentions AG_ARGS_FIT and explains what could be found in the  `params_aux` it updates.

Reply:
added